Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Copy content using te role of Editor
+    In order to validate the copy action
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
+        Given a "Destiny" folder exists
+        And a "News Flash" article exists
+        When I copy the "News Flash" into "Destiny" folder
```

Comment:
That was the first approach, but I simplified it due readability reasons and because the "Manipulate" option and respective sub-bar are not yet contemplated in the Action Bar. Currently we only have the direct option (Copy, Move, Send to Trash).

Maybe the first copy scenario might have full details from user's perspective, using current implementation. When the implementation is updated to use the "Manipulate" menu, then we update the bdd scenario.
But i'm unsure about having these details in *all* scenarios. If so, we risk having too many details that might difficult the readability and understood of the real purpose of the scenarios objectives.